extensions/value_generators/models/generators_test.py

```diff
@@ -32,8 +32,15 @@ def test_copier(self):
             {}, **{'value': '{{a}}', 'parse_with_jinja': False}), '{{a}}')
         self.assertEqual(generator.generate_value(
             {'a': 'b'}, **{'value': '{{a}}', 'parse_with_jinja': True}), 'b')
+        self.assertIn('<object-editor obj-type=', generator.get_html_template())
```

Comment:
> Here and below, you need to use an expected value that is specific to the value generator under consideration. Otherwise, it's not clear what you are testing -- the snippets you use would appear in pretty much any HTML/JS file.

This comment still applies. Use something that's specific to Copier or to RandomSelector, for all four assertions here. Otherwise the test is a bit meaningless -- for example, every JS file starts with a copyright statement.

Reply:
Done